rtpsender.go

```diff
@@ -199,9 +201,21 @@ func (r *RTPSender) AddEncoding(track TrackLocal) error {
 }
 
 func (r *RTPSender) addEncoding(track TrackLocal) {
+	ssrcFEC, ssrcRTX := SSRC(0), SSRC(0)
+
+	if r.api.mediaEngine.isRTXEnabled(r.kind, []RTPTransceiverDirection{RTPTransceiverDirectionSendonly}) {
+		ssrcRTX = SSRC(randutil.NewMathRandomGenerator().Uint32())
+	}
+
+	if r.api.mediaEngine.isFECEnabled(r.kind, []RTPTransceiverDirection{RTPTransceiverDirectionSendonly}) {
+		ssrcFEC = SSRC(randutil.NewMathRandomGenerator().Uint32())
+	}
+
 	trackEncoding := &trackEncoding{
-		track: track,
-		ssrc:  SSRC(randutil.NewMathRandomGenerator().Uint32()),
+		track:   track,
+		ssrc:    SSRC(randutil.NewMathRandomGenerator().Uint32()),
```

Comment:
How about creating just one MathRandomGenerator and using it multiple times rather than creating a new one every time?